From b3e0dbdf653c4056cd02a50775c6da279d65f013 Mon Sep 17 00:00:00 2001
From: Dave Airlie <airlied@redhat.com>
Date: Tue, 10 Jan 2017 08:35:47 +1000
Subject: [PATCH 0412/2094] drm/amd/display: drop min/max wrappers

These aren't needed, and aren't really used in too many places.

Change-Id: I1a52c694d6d8a84f5756fed4f60f295173cba6b5
Signed-off-by: Dave Airlie <airlied@redhat.com>
Reviewed-by: Tony Cheng <Tony.Cheng@amd.com>
Reviewed-by: Harry Wentland <Harry.Wentland@amd.com>
Reviewed-by: Edward O'Callaghan <funfunctor@folklore1984.net>
---
 drivers/gpu/drm/amd/display/dc/core/dc.c        | 2 +-
 drivers/gpu/drm/amd/display/dc/dce/dce_clocks.c | 4 ++--
 drivers/gpu/drm/amd/display/dc/os_types.h       | 2 --
 3 files changed, 3 insertions(+), 5 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/dc/core/dc.c b/drivers/gpu/drm/amd/display/dc/core/dc.c
index 2277f9b..a734857 100644
--- a/drivers/gpu/drm/amd/display/dc/core/dc.c
+++ b/drivers/gpu/drm/amd/display/dc/core/dc.c
@@ -634,7 +634,7 @@ struct dc *dc_create(const struct dc_init_data *init_params)
 	full_pipe_count = core_dc->res_pool->pipe_count;
 	if (core_dc->res_pool->underlay_pipe_index >= 0)
 		full_pipe_count--;
-	core_dc->public.caps.max_targets = dm_min(
+	core_dc->public.caps.max_targets = min(
 			full_pipe_count,
 			core_dc->res_pool->stream_enc_count);
 
diff --git a/drivers/gpu/drm/amd/display/dc/dce/dce_clocks.c b/drivers/gpu/drm/amd/display/dc/dce/dce_clocks.c
index 2626120..dd922bd 100644
--- a/drivers/gpu/drm/amd/display/dc/dce/dce_clocks.c
+++ b/drivers/gpu/drm/amd/display/dc/dce/dce_clocks.c
@@ -289,7 +289,7 @@ static void dce_set_clock(
 
 	/* Make sure requested clock isn't lower than minimum threshold*/
 	if (requested_clk_khz > 0)
-		requested_clk_khz = dm_max(requested_clk_khz,
+		requested_clk_khz = max(requested_clk_khz,
 				clk_dce->dentist_vco_freq_khz / 64);
 
 	/* Prepare to program display clock*/
@@ -364,7 +364,7 @@ static void dce112_set_clock(
 
 	/* Make sure requested clock isn't lower than minimum threshold*/
 	if (requested_clk_khz > 0)
-		requested_clk_khz = dm_max(requested_clk_khz,
+		requested_clk_khz = max(requested_clk_khz,
 				clk_dce->dentist_vco_freq_khz / 62);
 
 	dce_clk_params.target_clock_frequency = requested_clk_khz;
diff --git a/drivers/gpu/drm/amd/display/dc/os_types.h b/drivers/gpu/drm/amd/display/dc/os_types.h
index 7f0c282..27ed2a6 100644
--- a/drivers/gpu/drm/amd/display/dc/os_types.h
+++ b/drivers/gpu/drm/amd/display/dc/os_types.h
@@ -52,8 +52,6 @@
 
 #define dm_vlog(fmt, args) vprintk(fmt, args)
 
-#define dm_min(x, y) min(x, y)
-#define dm_max(x, y) max(x, y)
 #endif
 
 
-- 
2.7.4

